javav2/example_code/ecs/Readme.md

```diff
@@ -0,0 +1,80 @@
+# Amazon Elastic Container Service Java code examples
+
+This README discusses how to run and test the Java code examples for Amazon Elastic Container Service.
+
+## Running the Amazon Elastic Container Service Java files
+
+**IMPORTANT**
+
+The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.
+
+Some of these examples perform *destructive* operations on AWS resources, such as deleting an Amazon ECS service. **Be very careful** when running an operation that deletes or modifies AWS resources in your account. It's best to create separate test-only resources when experimenting with these examples.
+
+To run these examples, you can setup your development environment to use Apache Maven or Gradle to configure and build AWS SDK for Java projects. For more information, 
+see [Get started with the AWS SDK for Java 2.x](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html). 
+
+
+ ## Testing the Amazon Elastic Container Service files
+
+You can test the Java code examples for Amazon Elastic Container Service by running a test file named **EcsTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+You can execute the JUnit tests from a Java IDE, such as IntelliJ, or from the command line by using Maven. As each test is ran, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+
+	Test 3 passed
+
+**WARNING**: _Running these JUnit tests manipulates real Amazon resources and may incur charges on your account._
+
+ ### Properties file
+Before running the Amazon Elastic Container Service JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to execute the JUnit tests. For example, you define an cluster name. If you do not define all values, the JUnit tests fail.
```

Comment:
Amazon Elastic Container Service -> Amazon ECS

you can define an cluster name -> you can define a cluster name

Reply:
Fixed